cdk/package.json

```diff
@@ -17,21 +17,19 @@
     "diff": "cdk diff --path-metadata false --version-reporting false"
   },
   "devDependencies": {
-    "@aws-cdk/assert": "1.147.0",
     "@guardian/eslint-config-typescript": "^0.7.0",
     "@types/jest": "^27.0.2",
     "@types/node": "16.11.7",
-    "aws-cdk": "1.147.0",
+    "aws-cdk": "2.59.0",
+    "aws-cdk-lib": "2.59.0",
+    "constructs": "10.1.215",
     "eslint": "^7.32.0",
     "jest": "^27.3.1",
     "prettier": "^2.4.1",
     "ts-jest": "^27.0.7",
     "ts-node": "^10.4.0",
     "typescript": "~4.4.3",
-    "@aws-cdk/core": "1.147.0",
-    "@aws-cdk/aws-sns-subscriptions": "1.147.0",
-    "@aws-cdk/aws-cloudwatch": "1.147.0",
-    "@guardian/cdk": "39.0.0",
+    "@guardian/cdk": "49.1.0",
```

Comment:
[49.2.0 is out](https://github.com/guardian/cdk/releases/tag/v49.2.0), it should be a no-up to update in this project.